apps/base-docs/docs/pages/identity/basenames/basenames-faq.mdx

```diff
@@ -1,3 +1,5 @@
+import { Callout } from 'vocs/components'
+
 ---
 title: Basenames FAQ
 slug: /tools/basenames-faq
```

Comment:
something wonky with the title here, maybe because the import is above the metadata

![image](https://github.com/user-attachments/assets/c1a9d8d8-5cd0-42a5-baf6-33ae2ac39ad3)